Data/DataModel/Event.cs

```diff
@@ -29,19 +29,70 @@ public Uri URL
         public bool IsInternEvent { get; set; }
         public DateTime TeamRegistrationBegin { get; set; }
         public DateTime TeamRegistrationEnd { get; set; }
+
+        /// <summary>
+        /// Returns whether or not team registration is active.
+        /// </summary>
+        /// <returns>True if the current date is between the team registration begin and end times.</returns>
+        public bool IsTeamRegistrationActive()
+        {
+            return DateTime.Now.CompareTo(TeamRegistrationBegin) > 0 &&
```

Comment:
We should be using UtcNow instead of Now.

Reply:
The long term plan is to handle this differently, but for now go with UtcNow so that's consistent.